From a60d7b81674449d1753fcf8bb15c57fc178424e6 Mon Sep 17 00:00:00 2001
From: Robby Workman <rworkman@slackware.com>
Date: Thu, 19 Feb 2015 19:36:21 -0600
Subject: [PATCH 1/2] Use $(datadir)/xfce4/mixer/ as MIXER_DATADIR

Follow common practice of other Xfce componenets and use
$(datadir)/xfce4/mixer/ instead of $(datadir)/xfce4-mixer/.
---
 icons/16x16/Makefile.am    | 2 +-
 icons/scalable/Makefile.am | 2 +-
 libxfce4mixer/Makefile.am  | 2 +-
 xfce4-mixer/Makefile.am    | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/icons/16x16/Makefile.am b/icons/16x16/Makefile.am
index 803a2e4..ace4a07 100644
--- a/icons/16x16/Makefile.am
+++ b/icons/16x16/Makefile.am
@@ -1,4 +1,4 @@
-iconsdir = $(pkgdatadir)/icons/hicolor/16x16/status
+iconsdir = $(datadir)/xfce4/mixer/icons/hicolor/16x16/status
 icons_DATA = audio-input-microphone-muted.png
 
 EXTRA_DIST = $(icons_DATA)
diff --git a/icons/scalable/Makefile.am b/icons/scalable/Makefile.am
index 5bfc12c..5212e63 100644
--- a/icons/scalable/Makefile.am
+++ b/icons/scalable/Makefile.am
@@ -1,4 +1,4 @@
-iconsdir = $(pkgdatadir)/icons/hicolor/scalable/status
+iconsdir = $(datadir)/xfce4/mixer/icons/hicolor/scalable/status
 icons_DATA = audio-input-microphone-muted.svg
 
 EXTRA_DIST = $(icons_DATA)
diff --git a/libxfce4mixer/Makefile.am b/libxfce4mixer/Makefile.am
index 792a95e..e864ea8 100644
--- a/libxfce4mixer/Makefile.am
+++ b/libxfce4mixer/Makefile.am
@@ -25,7 +25,7 @@ libxfce4mixer_la_CPPFLAGS = 						\
 	-DDATADIR=\"$(datadir)\"					\
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\"				\
 	-DG_LOG_DOMAIN=\"libxfce4mixer\"				\
-	-DMIXER_DATADIR=\"$(pkgdatadir)\"
+	-DMIXER_DATADIR=\"$(datadir)/xfce4/mixer\"
 
 libxfce4mixer_la_CFLAGS = 						\
 	$(PLATFORM_CFLAGS)						\
diff --git a/xfce4-mixer/Makefile.am b/xfce4-mixer/Makefile.am
index 4af69e9..f979b92 100644
--- a/xfce4-mixer/Makefile.am
+++ b/xfce4-mixer/Makefile.am
@@ -23,7 +23,7 @@ xfce4_mixer_CPPFLAGS = 							\
 	-I$(top_srcdir)							\
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\"				\
 	-DDATADIR=\"$(datadir)\"					\
-	-DMIXER_DATADIR=\"$(pkgdatadir)\"				\
+	-DMIXER_DATADIR=\"$(datadir)/xfce4/mixer\"		\
 	-DG_LOG_DOMAIN=\"xfce4-mixer\"
 
 xfce4_mixer_CFLAGS = 							\
-- 
2.3.1

